server/src/models/User.ts

```diff
@@ -17,3 +19,15 @@ export interface DisplayUser {
   readonly name: string;
   readonly imageUrl?: string;
 }
+
+export const CreateUserSchema = z.object({
+  email: z.email(),
+  name: z.string(),
+  imageUrl: z.string().optional(),
+  isActive: z.boolean().default(true),
+});
+
+export const UpdateUserSchema = CreateUserSchema.partial();
```

Comment:
Update user should require the same fields like when creating a user. Any reason why we want this partial?

Reply:
As long as it will not allow to accidentally update a user with `null` email or similar - its ok by me